### PR TITLE
Add new port R&Watch-Dobtopus

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { R&Watch-Dobtopus }

## Game Information
- **Title**: { R&Watch-Dobtopus }
- **URL**: { https//:Nizakashii.itch.io/R&Watch }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] JELOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/rwatch/port.json
Comment (Cebion) [inline ports/rwatch/port.json:1]: Port.json wrong

foldername and zip name need to be the same name. Also .zip is missing

Comment (Cebion) [reply ports/rwatch/port.json:1]: The port.json is also from a installed setup

File: ports/rwatch/README.md
Comment (Cebion) [inline ports/rwatch/README.md:1]: The url to the game has to be a proper Internet link so https://url not @username

**Review by Cebion — CHANGES_REQUESTED** [review]
See comments

File: ports/rwatch/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Notes
+
+Huge thank you to the creator of this game, [Nizakashii] (@Nizakashii.itch.io) and thanks to all who tested and reviewed this port.
+-Download (@Nizakashii.itch.io/R&Watch-Dobtopus)
+-Extract and place data.win file into gamedata folder
```
Comment (Slayer366) [inline ports/rwatch/README.md:5]: Like this:
```suggestion
Huge thank you to the creator of this game, nizakashii@itch.io, and thanks to all who tested and reviewed this port.
-Download from [Nizakashii] (https://nizakashii.itch.io/rwatch)
-Extract and place data.win file into gamedata folder
```

**Review by Slayer366 — CHANGES_REQUESTED** [review]

File: ports/rwatch/port.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "items_opt": [],
+  "attr": {
+    "title": "R&Watch-Dobtopus",
+    "porter": [
+      "MadShmupper",
+      "Jeod",
+      "JanTrueno"
+    ],
+    "desc": "A tight boss rush that's a challenge without being too brutal. The graphics have a sleek minimalist style with loads of visual flairs. ",
+    "inst": "-Download at (https//:nizakashii.itch.io/R&Watch\n-Extract and place data.win into gamedata folder",
```
Comment (Slayer366) [inline ports/rwatch/port.json:17]: Missing a closing parentheses ')' after the internet link

**Review by Slayer366 — CHANGES_REQUESTED** [review]

**Comment by Cebion** [comment]
One last question
"porter": [
      "MadShmupper",
      "Jeod",
      "JanTrueno"
    ],
Did really all 3 work actively on the port and can answer questions about the port?
Or did one just help test or zip it up
Because they will be asked if anything is wrong iwth the port in the future
Else they'll just get a thank you note on the readme

**Review by Cebion — APPROVED** [review]

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**